adminSiteClient/CreateDataInsightModal.tsx

```diff
@@ -790,6 +900,30 @@ function DataInsightCreationFeedback({
     )
 }
 
+function TopicTagsFeedback({
+    topicTags,
+    progress,
+}: {
+    topicTags: MinimalTag[]
+    progress: Progress
+}) {
+    if (topicTags.length === 0) return null
+    return (
+        <li>
+            <span>
+                Tagging the newly created data insight, with{" "}
+                {topicTags.map((tag, index) => (
+                    <Fragment key={tag.id}>
+                        <i>{tag.name}</i>
+                        {index < topicTags.length - 1 && " and "}
```

Comment:
I'm surprised we don't already have a util called `englishJoin` or something for this